groups/api/http/decode_test.go

```diff
@@ -14,10 +14,13 @@ import (
 	api "github.com/absmach/supermq/api/http"
 	apiutil "github.com/absmach/supermq/api/http/util"
 	"github.com/absmach/supermq/groups"
+	"github.com/absmach/supermq/internal/nullable"
 	"github.com/absmach/supermq/pkg/errors"
 	"github.com/stretchr/testify/assert"
 )
 
+var description = "random"
```

Comment:
Description is already defined in `endpoints_test.go` Why we could not reuse the same in this file ?